API.md

```diff
@@ -460,7 +454,7 @@ When the server contains exactly one connection, `registrations` is an object wh
 registered plugin name and value contains:
 - `version` - the plugin version.
 - `name` - the plugin name.
-- `options` - options used to register the plugin.
+- `options` - options object passed to the plugin during registration.
```

Comment:
@AdriVanHoudt it does the options are part of the object with the `register` key ["an object with the following: ..."](https://hapijs.com/api#serverregisterplugins-options-callback).

    var server = new hapi.Server();
    server.connection({port: 3000})

    var pluginFunction = function(server, optionsPassedToPlugin, nextFn) {
        console.log('pluginFunction called optionsPassedToPlugin:', optionsPassedToPlugin);
        nextFn();
    }
    pluginFunction.attributes = {
        name: 'pluginFunction',
    }

    var options = {
        once: true, // ignored in terms of actual `once` option
        something: 'else',
    };
    var callback = function(err) {
        console.log('callback called')
    };

    server.register({
        register: pluginFunction,
        options: options,
    }, callback)

    console.log(server.registrations)

    // Output:
    //
    // pluginFunction called optionsPassedToPlugin: { once: true, something: 'else' }
    // { pluginFunction:
    //    { version: '0.0.0',
    //      name: 'pluginFunction',
    //      options: { once: true, something: 'else' },
    //      attributes:
    //       { name: 'pluginFunction',
    //         pkg: [Object],
    //         multiple: false,
    //         connections: true } } }
    // callback called

Did I understand your question correctly?